README.md

```diff
@@ -21,8 +21,9 @@ For information about mod development, check out the [Everest Wiki](https://gith
 - ***If you just want to install Everest, go to the [Everest website](https://everestapi.github.io/).***
 - If you've **previously installed Everest and updated Celeste** or switched betas / branches, delete the `orig` directory where Celeste is installed.
     - macOS: Right-click and browse the Celeste app in Finder, then navigate to `Contents`, then `MacOS`.
-- Clone the Everest repo **+ submodules**, either in your IDE, via the CLI, or by downloading the .zip from GitHub.
+- Clone the Everest repo **+ submodules**, either in your IDE or via the CLI.
 - Restore Nuget packages either via your IDE or the command line.
+- Everest requires both 6.0 and 7.0+ versions of the .NET Core SDK for the build process.
```

Comment:
This is actually a lie and i should've specified in my message on the issue.
You need [net 7.0.200](https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#lifecycle) or higher to build the MonoMod solution (so the SDK) but the net 6.0 is only used to *run* a binary in the compilation process, so the net6.0 runtime is enough.

The exact net 7 version required to build could be a really nice addition but the net 6.0 runtime/SDK deal is so minor i don't think its worth mentioning. 
Just wanted to clear things up :)

Reply:
Thanks, I'll add a commit that clarifies this.